vowpalwabbit/CMakeLists.txt

```diff
@@ -187,5 +187,5 @@ if(VW_INSTALL)
   # VW target headers
   install(
     FILES ${vw_all_headers} ${CMAKE_CURRENT_BINARY_DIR}/config.h
-    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/vowpalwabbit)
```

Comment:
I am a little wary here, because that would mean that a version upgrade could break people's makefiles. Probably not a huge deal, particularly with pkgconfig / CMake (especially if we add a find macro module), but let's make sure we are doing this with deliberation.

Reply:
So in 8.6.1 this is actually the behavior that existed. This is a fix.

Reply:
Ah, I got it. Could you add that to the commit description to explain we are backing out an change in 8.7?